docs/source/methods/method_of_characteristics.rst

```diff
@@ -274,7 +274,7 @@ Another common approximation for MOC is to assume that the source :math:`Q_g` is
 The Constant Cross-Section Approximation
 ========================================
 
-In addition to the flat source approximation, it is assumed that the material properties are constant across each FSR. The area-averaged cross-sections for FSR :math:`i \in \{1, 2, ..., I\}` with area :math:`A_{i}` are defined as:
+In addition to the flat source approximation, it is assumed that the material properties are constant across each SR. The area-averaged cross-sections for SR :math:`i \in \{1, 2, ..., I\}` with area :math:`A_{i}` are defined as:
```

Comment:
I guess I'm not really sure how you want to refactor this documentation, but it appears you may have searched for "flat source region" but may also want to look for "flat source approximation," if we deem it necessary to change the terminology here (or instead add a second section on linear source approximation).